br/pkg/streamhelper/regioniter_test.go

```diff
@@ -81,6 +82,10 @@ func (c constantRegions) BlockGCUntil(ctx context.Context, at uint64) (uint64, e
 	return 0, status.Error(codes.Unimplemented, "Unsupported operation")
 }
 
+func (c constantRegions) FetchCurrentTS(ctx context.Context) (uint64, error) {
+	return uint64(time.Now().Unix()), nil
```

Comment:
For making it an TSO, you may also need to append it with a "logical part" and shift the physical unix millisecond left. Read [this](https://docs.pingcap.com/tidb/stable/tso#timestamp-oracle-tso-in-tidb) for more details.

TL;DR: You may directly use `oracle.ComposeTS(time.Now().UnixMilli(), 0)` here.